features/login.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Login page
+  As an User
+  In order to access my account
+  I need to be able to login
+
+  Background:
+    Given I am on the "login" page
+    And the following users exist
+    | email                | password    |
+    | orphanage@random.com | my_password |
+
+  Scenario: Accessing the login page
```

Comment:
@MikaelFeher This scenario is implicitly tested in all the other scenarios -- I think you can remove this as it's redundant.